include/singa/utils/updater.h

```diff
@@ -87,6 +90,12 @@ class InvTLRGen : public LRGenerator {
  */
 class Updater {
  public:
+
+  /* added for python binding */
+  static Updater* CreateUpdater(const string str);
+  static void UpdateParams(Updater* updater, Layer* layer, int step);
```

Comment:
Do we have a python class for Param?
If yes, then we do not need the UpdateParams function..